Userland/Games/Chess/main.cpp

```diff
@@ -138,7 +138,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     auto board_theme_menu = TRY(style_menu->try_add_submenu("Board Theme"));
     board_theme_menu->set_icon(Gfx::Bitmap::try_load_from_file("/res/icons/chess/mini-board.png").release_value_but_fixme_should_propagate_errors());
 
-    for (auto& theme : Vector({ "Beige", "Green", "Blue" })) {
+    for (auto& theme : { "Beige", "Green", "Blue" }) {
```

Comment:
Since you marked the others loop variables in this commit as `const` maybe do this one too?

Reply:
Right, I forgot that one.